src/Apps/Artwork/Components/ArtworkSidebar/index.tsx

```diff
@@ -16,13 +17,25 @@ import { ArtworkSidebar_artwork } from "__generated__/ArtworkSidebar_artwork.gra
 import { ArtworkSidebarQuery } from "__generated__/ArtworkSidebarQuery.graphql"
 import { SystemContext } from "Artsy"
 import { SystemQueryRenderer as QueryRenderer } from "Artsy/Relay/SystemQueryRenderer"
+import { SecurePaymentFragmentContainer as SecurePayment } from "../TrustSignals/SecurePayment"
 
 export interface ArtworkSidebarProps {
   artwork: ArtworkSidebar_artwork
 }
 
 const ArtworkSidebarContainer = Box
 
+const TrustSignalsContainer = styled.div`
```

Comment:
Overall this looks great!

Out of curiosity, why do we need this CSS vs. using conditional logic/adding spacers directly in our components?

Reply:
It's mainly due to the complex display states of these components paired with their spacing needs. 

The components above have 30px mb and the separator below has nothing on the top. If any one of these trust signals are rendered, then we have to attach a 30px mb to ensure the separator isn't jutting up against the components. We can't just add mb to the trust signals though, because they may and or may not render and the trust signals are spaced out by 20px from each other.

This was the easiest way to preserve the rendering relationships and spacing.

Reply:
Ah gotcha. I guess if this top-level component knew which of the child components would be rendered then it could use a `Join`, but that does seem like it would make the logic more complex.

Reply:
Yeah, I used join to start with and thought about making a PR for it to serve before/after components, but this seems more straight forward.